src/orion/benchmark/task/hpobench.py

```diff
@@ -0,0 +1,86 @@
+"""
+Task for HPOBench
+=================
+"""
+import importlib
+import subprocess
+from typing import Dict, List, Union
+
+from orion.algo.space.configspace import to_orionspace
+from orion.benchmark.task.base import BenchmarkTask
+
+
+class HPOBench(BenchmarkTask):
+    """Benchmark Task wrapper over HPOBench (https://github.com/automl/HPOBench)
+
+    For more information on HPOBench, see original paper at https://arxiv.org/abs/2109.06716.
+
+    Katharina Eggensperger, Philipp Müller, Neeratyoy Mallik, Matthias Feurer, René Sass, Aaron Klein,
+    Noor Awad, Marius Lindauer, Frank Hutter. "HPOBench: A Collection of Reproducible Multi-Fidelity
+    Benchmark Problems for HPO" Thirty-fifth Conference on Neural Information Processing Systems
+    Datasets and Benchmarks Track (Round 2).
+
+    Parameters
+    ----------
+    max_trials : int
+        Maximum number of trials for this task.
+    hpo_benchmark_class : str
+        Full path to a particular class of benchmark in HPOBench.
+    benchmark_kwargs: str
+        Optional parameters to create benchmark instance of class `hpo_benchmark_class`.
+    objective_function_kwargs: dict
+        Optional parameters to use when calling `objective_function` of the benchmark instance.
+    """
+
+    def __init__(
+        self,
+        max_trials: int,
+        hpo_benchmark_class: Union[str, None] = None,
```

Comment:
Should it support `None` as default? Or maybe `''` by default to simplify typing.

Reply:
wow, I forgot what I was thinking ^^